ui/packages/consul-ui/tests/acceptance/steps/dc/intentions/read-only-steps.js

```diff
@@ -0,0 +1,10 @@
+import steps from '../../steps';
+
+// step definitions that are shared between features should be moved to the
+// tests/acceptance/steps/steps.js file
+
+export default function(assert) {
+  return steps(assert).then('I should find a file', function() {
+    assert.ok(true, this.step);
+  });
+}
```

Comment:
Did you end up using this step? Looks like it always passes?

Reply:
Ah this is where it's just lack of context into the addon we use for our acceptance tests.

The addon _requires_ these files in order to run (the idea is you write a entire set of steps for every test, which seems like a lot of repetitive work). Instead we have about 25-30 reusable steps that we use for all our tests. Unfortunately the addon _requires_ these files to still exist and generates them exactly like this for each and every test. If you delete them it breaks. Therefore we have oodles of these files in a nested folder structure that are all exactly the same. It's annoying, but I just pretend they aren't there.

We had a PR a while ago to stop it from doing this which made it much nicer, but unfortunately it didn't go in and has been the root of confusion both previous to that PR and ever since 😹

Reply:
Yikes, that isn’t ideal. Still, explanation makes sense for the context of this PR.